overlord/ifacestate/handlers.go

```diff
@@ -1532,6 +1534,120 @@ func (m *InterfaceManager) doHotplugDisconnect(task *state.Task, _ *tomb.Tomb) e
 	return nil
 }
 
+func (m *InterfaceManager) doHotplugAddSlot(task *state.Task, _ *tomb.Tomb) error {
+	st := task.State()
+	st.Lock()
+	defer st.Unlock()
+
+	coreSnapInfo, err := snapstate.CoreInfo(st)
```

Comment:
this needs to find the system snap, not only core, no?

Reply:
Fixed.